Gemfile.lock

```diff
@@ -35,7 +41,10 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+  bridge-pbn!
```

Comment:
By the way, @danielevans , what is this?  Why does it have a gemfile lock on itself?